frontend-react/src/components/Table/TableFilters.test.tsx

```diff
@@ -11,6 +11,8 @@ describe("Rendering", () => {
     beforeEach(() => {
         renderApp(
             <TableFilters
+                startDateLabel="From (Start Range):"
```

Comment:
`From (Start Range):` and `Until (End Range):` are repeated six times. can we either make them a default somewhere or declare some `enum` we can export and re-use?